### PR TITLE
create symlink for /usr/bin/python

### LINKED ISSUE #77: run ansible playbook against hosts with python3 installed by default
**Describe the bug**
In Ubuntu 16.04 and later, python3 is installed by default and python2 is removed, this leads to missing default python interpreter used by ansible `/usr/bin/python` on such hosts and the play is failed with message:
```
bash: /usr/bin/python: No such file or directory
```
**Version**
Latest (or any) version of `vpnc-deployer-cli`

**To Reproduce**
Run the ansible playbook against any linux distro with python3 installed by default, such as: Ubuntu 18.04, Ubuntu 20.04...

**Expected behavior**
The ansible playbook should be able to resolve this scenario

### PR DESCRIPTION
**Pull request by anhcq151**
close #77 
Method 2 is chosen in this PR

### CODE REVIEW

File: cli/ansible/wf-ensure-python.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+- name: Ensure /usr/bin/python path exists
+  hosts: all
+  gather_facts: false
+
+  tasks:
+    - name: symlink /usr/bin/python -> /usr/bin/python3
```
Comment (zero88) [inline cli/ansible/wf-ensure-python.yml]: symlink python3 -> python?

File: cli/ansible/wf-vpnc-setup.yml [hunk 2]
```diff
@@ -1,3 +1,5 @@
+- import_playbook: wf-ensure-python.yml
```
Comment (zero88) [inline cli/ansible/wf-vpnc-setup.yml]: Remove this line
`wf-ensure-python.yml` should be on demand. And in future, any ansible environment setup will not be in any specific deployment app bundle

**Review by zero88 — CHANGES_REQUESTED** [review]

File: cli/ansible/wf-vpnc-setup.yml (hunk 2 above)
Comment (anhcq151) [reply cli/ansible/wf-vpnc-setup.yml]: This task is quite safe on any hosts and there should be no impact to target environment. I still think it's safe to keep it on top of the playbook.

Comment (zero88) [reply cli/ansible/wf-vpnc-setup.yml]: @anhcq151 It is not related to safe or no safe.
I emphasis once again: VPN is one of many applications will be deployed on target environment. Keep it clean and single responsibility only and only for VPN. Don't touch any general configuration on environment that doesn't relate directly to VPN task.
Tweak Ansible/Python environment for remote deployment is unrelated to VPN task. This task will be in common tasks in Ansible base image.

Just simple keep workflow in this repository until I provide Ansible base image, and make a call manually in first time of your test environment.

**Review by zero88 — CHANGES_REQUESTED** [review]
See above message

**Review by zero88 — APPROVED** [review]

**Pull request merged**